docs/setup/install/rpm.asciidoc

```diff
@@ -153,12 +153,7 @@ https://artifacts.elastic.co/downloads/kibana/kibana-oss-{version}-x86_64.rpm
 
 endif::[]
 
-==== SysV `init` vs `systemd`
-
-include::init-systemd.asciidoc[]
-
-[[rpm-running-init]]
-include::rpm-init.asciidoc[]
+==== `systemd`
```

Comment:
I think we should actually remove this line.

Reply:
Looks a little odd and the next header covers it:

![image](https://user-images.githubusercontent.com/40265/97504855-fedfb680-1934-11eb-9410-5f42a79915b0.png)